scripts/main.js

```diff
@@ -71,4 +72,74 @@ const departments = {
     }
 }
 
-console.log(departments);
\ No newline at end of file
+console.log(departments);
+// //opdracht1a
+// console.log('De afdeling Sales heeft' + ' ' + [departments.sales.numberOfEmployees] + ' '  + 'medewerkers' + '.')
+// //opdracht1b
+// console.log('Marketing is een leuke afdeling om te werken.' + ' ' +  [departments.marketing.description] + '.')
+// //opdracht1c
+// console.log('De afdeling Customer Service heeft' + ' ' + [departments["customer-service"].numberOfEmployees] + ' ' + 'medewerkers' + '.')
+// //opdracht1d
+// console.log('Sales is een uitdagende afdeling om te werken als Verkoopmanager.' + ' ' + departments.sales.jobs[1].description)
+
+
+
+//opdracht 2a en opdracht2b
+const userInputOpdracht2 = prompt('Over welke afdeling wil je meer informatie? Kies uit:' + '[marketing / sales / customer-service]');
+console.log(userInputOpdracht2);
+
+console.log('Begin of switch for opdracht2')
+switch (userInputOpdracht2) {
+    case 'marketing':
+        console.log('Je koos' + ' ' + userInputOpdracht2 + '. ' + departments[userInputOpdracht2].description)
+        break
+    case 'sales':
+        console.log('Je koos' + ' ' + userInputOpdracht2 + '. ' + departments[userInputOpdracht2].description)
+        break
+    case 'customer-service':
+        console.log('Je koos' + ' ' + userInputOpdracht2 + '. ' + departments[userInputOpdracht2].description)
+        break
+
+//opdracht2c
+    default:
+        console.error('Hmm, ik heb geen geldige keuze van je ontvangen. Klopt dat? Probeer maar eens opnieuw. Je mag opgeven: [marketing / sales / customer-service]. Ververs de pagina maar.')
+}
+console.log('End of switch for opdracht 2')
+
+
+console.log('Begin of switch for opdracht3')
+
+
+
+let userInputOpdracht3;
+let userInputAnswerOpdracht3 = prompt('Je koos' + ' ' +  userInputOpdracht2 + '.' + ' ' +  'Over welke functie wil je meer weten? Voer een getal in tussen 0 en 3:\n\n' + '0:' + departments[userInputOpdracht2].jobs[0].title + '\n' + '1:' + departments[userInputOpdracht2].jobs[1].title + '\n' + '2:' + departments[userInputOpdracht2].jobs[2].title + '\n' + '3:' + departments[userInputOpdracht2].jobs[3].title);
```

Comment:
het is jou wel gelukt om die hele message in 1 keer in de prompt te krijgen, nice! (ik had er een aparte string-variabele voor nodig )